weblab/entities/tests/test_views.py

```diff
@@ -51,6 +51,23 @@ def test_create_model(self, logged_in_user, client, helpers):
 
         assert entity.repo_abs_path.exists()
 
+    def test_create_model_same_name(self, logged_in_user, other_user, client, helpers):
```

Comment:
I don't think this is checking anything new at present, as it doesn't create two models? We probably want to defer the fixing of the bug on entity creation for another PR, so I'd suggest removing this test for now.